config/hashing.go

```diff
@@ -13,8 +13,18 @@ func init() {
 		// by the framework hash facade.
 		// Default driver is "argon2id", because it is the most secure.
 		//
-		// Supported Drivers: "argon2id", "bcrypt"
-		"driver": "argon2id",
+		// Supported Drivers: "bcrypt", "argon2id"
+		"driver": "bcrypt",
```

Comment:
The `argon2id` driver will spend more time(about 200ms) when `facades.Hash.Make`, I found the default value is `bcrypt` in Laravel, so I optimize this. Please help confirm @DevHaoZi

Reply:
> The `argon2id` driver will spend more time(about 200ms) when `facades.Hash.Make`, I found the default value is `bcrypt` in Laravel, so I optimize this. Please help confirm @DevHaoZi

Yes, Laravel default use bcrypt, I chose argon2id because it's modern and more secure.

Reply:
Yes, I think so, user can use argon2id if need more secure(don't mind the speed).

Reply:
> Yes, I think so, user can use argon2id if need more secure(don't mind the speed).

ok, please also modify the description in the header of the hashing.go file.